src/components/EasyDropdown/index.js

```diff
@@ -159,7 +159,7 @@ EasyDropdown.propTypes = {
       onClick: PropTypes.func,
     })
   ),
-  menuProps: PropTypes.shape({ ...Menu.propTypes }),
+  menuProps: PropTypes.object,
```

Comment:
Is this component coming up for TS-ification soon?

Reply:
Yes, I have a branch for it I believe... It's coming, I promise